R/Filtering.R

```diff
@@ -502,3 +502,251 @@ FilterParFieldReview <- function(field, .data, parameter) {
+             "DetectionQuantitationLimitTypeName",
+             "LabSamplePreparationUrl",
+             "ProviderName",
+             "MonitoringLocationTypeName",
+             "MonitoringLocationDescriptionText",
+             "HUCEightDigitCode",
+             "LatitudeMeasure",
+             "LongitudeMeasure",
+             "SourceMapScaleNumeric",
+             "HorizontalCollectionMethodName",
+             "HorizontalCoordinateReferenceSystemDatumName",
+             "VerticalCollectionMethodName",
+             "VerticalCoordinateReferenceSystemDatumName",
+             "CountryCode",
+             "StateCode",
+             "CountyCode",
+             "AquiferName",
+             "LocalAqfrName",
+             "FormationTypeText",
+             "AquiferTypeName",
+             "ConstructionDateText",
+             "MethodSpecificationName",
+             "ProjectDescriptionText",
+             "SamplingDesignTypeCode",
+             "QAPPApprovedIndicator",
+             "QAPPApprovalAgencyName",
+             "ProjectFileUrl",
+             "ProjectMonitoringLocationWeightingUrl",
+             "TADA.LatitudeMeasure",
+             "TADA.LongitudeMeasure",
+             "TADA.InvalidCoordinates.Flag",
+             "TADA.ActivityMediaName", 
+             "TADA.CharacteristicName",
+             "TADA.CharacteristicGroup",
+             "CharacteristicNameUserSupplied",
+             "TADA.SuggestedCharacteristicName",
+             "TADA.CharacteristicNameAssumptions",
+             "TADA.TotalN_TotalP_CharacteristicNames_AfterSummation",
+             "TADA.TotalN_TotalP_Summation_Identifier",
+             "TADA.TotalN_TotalP_ComboLogic",
+             "TADA.AggregatedContinuousData.Flag",
+             "TADA.ResultMeasureValue",
+             "TADA.ResultMeasureValueDataTypes.Flag",
+             "TADA.CensoredData.Flag",
+             "TADA.CensoredMethod",
+             "TADA.ResultUnit.Flag",
+             "TADA.MethodSpecificationName",
+             "TADA.AnalyticalMethod.Flag",
+             "TADA.MethodSpeciation.Flag",
+             "TADA.ResultSampleFractionText",
+             "TADA.SampleFraction.Flag",
+             "TADA.SuggestedSampleFraction",
+             "TADA.FractionAssumptions",
+             "TADA.ComparableDataIdentifier",
+             "TADA.RemoveReason"
+    )
+  }
+  if(display=="all"){
+    cols = names(df)
+  }
+  
+  df = df[,names(df)%in%cols]
+  
+  # CREATE LIST OF FIELDS
+  # Find count of unique values in each column
+  col.names <- data.frame(Count = apply(df, 2, function(x) length(unique(x))))
+  # Create "Fields" column from row names
+  col.names$Fields <- row.names(col.names)
+  # Remove row names
+  row.names(col.names) <- NULL
+  # Reorder columns
+  col.names <- col.names[, c(2, 1)]
+  
+  # Reorder Count column in col.names from largest to smallest number
+  col.names <- col.names %>%
+    dplyr::arrange(desc(Count))
+  
+  return(col.names)
+}
+
+
+#' Field Values Pie Chart
```

Comment:
since this is separated from the filter function now, does it make sense to move it to Figures.R?

Reply:
I am okay with retiring the old filter functions, but need to update vignette to use these instead